apps/frontend/src/app/PageArtifact/ArtifactEditor/Components/SubstatInput.tsx

```diff
@@ -37,6 +38,8 @@ export default function SubstatInput({ index, artifact, setSubstat }: { index: n
   if (!rollNum && key && value) error = error || t`editor.substat.error.noCalc`
   if (allowedRolls < 0) error = error || t("editor.substat.error.noOverRoll", { value: allowedRolls + rollNum })
 
+  const marks = useMemo(() => key ? artifactSubRolls(rarity, key).map(v => ({ value: v, })) : [{ value: 0 }], [key, rarity])

```

Comment:
Should we have a 0-state as well? Otherwise you can't select an initial roll with the 4-button style input, as the 4 buttons always start with the second roll
![image](https://user-images.githubusercontent.com/36019388/218946643-22b594d8-cd2b-40bc-8a42-96c12360916f.png)

Reply:
Yeah i was wondering about this. The issue is that the validation code is trying to match the min value, but the editor should allow for "non valid" values while user is editing. either i let validation allow a 0, or add a special cached value in the UI that allows user more freedom of editing(will get desynced from the actual underlying data representation)